echopype/tests/mask/test_mask.py

```diff
@@ -287,33 +291,84 @@ def test_check_source_Sv_freq_diff(n: int, n_chan_freq: int, add_chan: bool, add
 
     source_Sv = get_mock_freq_diff_data(n, n_chan_freq, add_chan, add_freq_nom)
 
-    _check_source_Sv_freq_diff(source_Sv, freqAB=freqAB, chanAB=chanAB)
+    _check_freq_diff_source_Sv(source_Sv, freqAB=freqAB, chanAB=chanAB)
 
 
 @pytest.mark.parametrize(
-    ("n", "n_chan_freq", "freqAB", "chanAB", "diff", "operator", "mask_truth"),
+    ("freqABEq", "chanABEq"),
     [
-        (5, 4, [1.0, 3.0], None, 1.0, "==", np.identity(5)),
-        (5, 4, None, ['chan1', 'chan3'], 1.0, "==", np.identity(5)),
-        (5, 4, [3.0, 1.0], None, 1.0, "==", np.zeros((5, 5))),
-        (5, 4, None, ['chan3', 'chan1'], 1.0, "==", np.zeros((5, 5))),
-        (5, 4, [1.0, 3.0], None, 1.0, ">=", np.identity(5)),
-        (5, 4, None, ['chan1', 'chan3'], 1.0, ">=", np.identity(5)),
-        (5, 4, [1.0, 3.0], None, 1.0, ">", np.zeros((5, 5))),
-        (5, 4, None, ['chan1', 'chan3'], 1.0, ">", np.zeros((5, 5))),
-        (5, 4, [1.0, 3.0], None, 1.0, "<=", np.ones((5, 5))),
-        (5, 4, None, ['chan1', 'chan3'], 1.0, "<=", np.ones((5, 5))),
-        (5, 4, [1.0, 3.0], None, 1.0, "<", np.ones((5, 5)) - np.identity(5)),
-        (5, 4, None, ['chan1', 'chan3'], 1.0, "<", np.ones((5, 5)) - np.identity(5)),
+        ("1.0Hz-2.0Hz==1.0dB", None),
+        (None, '"chan1"-"chan3"==1.0dB'),
+        ("1.0 kHz - 2.0 MHz>=1.0 dB", None),
+        (None, '"chan2-12 89" - "chan4 89-12" >= 1.0 dB'),
```

Comment:
These variations are great! I can see that the current test makes sure that `_parse_freq_diff_eq` is run through without problem, though the values are not checked. Could you add a section in the test for the parsed values, to make sure the output is correct? Thanks!